src/pages/data-feeds/rates-feeds/index.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: ../../../layouts/MainLayout.astro
+section: dataFeeds
+date: Last Modified
+title: "Rate and Volatility Feeds"
+isIndex: true
+permalink: "docs/data-feeds/nft-floor-price/"
+whatsnext:
+  {
+    "Learn how to read answers from Data Feeds": "/data-feeds/price-feeds/",
+    "Learn how to get Historical Price Data": "/data-feeds/historical-data/",
+    "Find contract addresses for Rate and Volatility  Feeds": "/data-feeds/rates-feeds/addresses/",
+    "Data Feeds API Reference": "/data-feeds/api-reference/",
+  }
+---
+
+Several feeds provide data for interest rates, interest rate curves, and asset volatility. You can read these feeds the same way that you read other Data Feeds. Specify the [Rate or Volatility Feed Address](/data-feeds/rates-feeds/addresses/) that you want to read instead of specifying a Price Feed address. See the [Using Data Feeds](/data-feeds/using-data-feeds/) page to learn more.
```

Comment:
```suggestion
Chainlink rate and volatility feeds provide data for interest rates, interest rate curves, and asset volatility. You can read these feeds the same way that you read other Data Feeds. Specify the [Rate or Volatility Feed Address](/data-feeds/rates-feeds/addresses/) that you want to read instead of specifying a Price Feed address. See the [Using Data Feeds](/data-feeds/using-data-feeds/) page to learn more.
```